backend/src/main/kotlin/app/ehrenamtskarte/backend/stores/utils/MapCsvToStore.kt

```diff
@@ -0,0 +1,11 @@
+package app.ehrenamtskarte.backend.stores.utils
+
+import app.ehrenamtskarte.backend.stores.COUNTRY_CODE
+import app.ehrenamtskarte.backend.stores.importer.common.types.AcceptingStore
+import app.ehrenamtskarte.backend.stores.webservice.schema.types.CSVAcceptingStore
+
+fun mapCsvToStore(csvStore: CSVAcceptingStore): AcceptingStore {
+    return AcceptingStore(
+        csvStore.name.clean()!!, COUNTRY_CODE, csvStore.location.clean()!!, csvStore.postalCode.clean()!!, csvStore.street.clean()!!, csvStore.houseNumber.clean()!!, "", csvStore.longitude!!.toDouble(), csvStore.latitude!!.toDouble(), csvStore.categoryId!!.toInt(), csvStore.email.clean()!!, csvStore.telephone.clean()!!, csvStore.homepage.clean()!!, csvStore.discountDE.orEmpty() + "\n\n" + csvStore.discountEN.orEmpty(), null, null
```

Comment:
if discountDE and discountEN are null, I think the expected description should be null, or?
currently it's "\n\n"

Reply:
yes i added a check

Reply:
now if discountDE is null and discountEN is not null ("test description", for example), it's saved like "\n\ntest description"

Reply:
addressed this in last commit by using a `mutableList` now its even better extendable if we want to add another language for discount someday